plsync/sites.py

```diff
@@ -157,7 +159,7 @@
     makesite('ord05','128.177.163.0',  '2001:438:fffd:2f::',   'Chicago_IL', 'US', 41.978600, -87.904700, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
     makesite('par02','212.73.231.192', '2001:4c08:2003:3f::',  'Paris', 'FR', 48.858400, 2.349010, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
     makesite('par03','80.239.222.64',  '2001:2030:35::',       'Paris', 'FR', 48.858400, 2.349010, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
-    makesite('par04','77.67.115.128',  '2001:668:1f:64::',     'Paris', 'FR', 48.858400, 2.349010, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
+    makesite('par04','77.67.119.129',  '2001:668:1f:6b::',     'Paris', 'FR', 48.858400, 2.349010, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
```

Comment:
The IPv4 subnet should be 77.67.119.128, right?

Reply:
yes. sorry.

Reply:
Is the IPv6 address correct per Mike's recent update to the [ticket](https://waffle.io/m-lab/ops-tracker/cards/5925814974550a0893f76a8c) @nkinkade ?

Reply:
Yeah, the IPv6 address looks correct, per what Mike dropped in [ops-tracker/issues/103](https://github.com/m-lab/ops-tracker/issues/103#issuecomment-329262286).

Reply:
Thanks @nkinkade - I just made another commit to correct that. PTAL?